crypto-ballerina/src/crypto/tests/unit-tests/hash-function-test.bal

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/test;
+
+@test:Config {}
+function testHashMd5() {
+    byte[] input = "Ballerina test".toBytes();
+    string expectedMd5Hash = "3B12196DB784CD9F86CC635D32764FDF".toLowerAscii();
+    test:assertEquals(crc32b(input), "d37b9692", msg = "Error while Hash with CRC32b.");
+}
+
```

Comment:
The test case for `crc32b1 API is missing here. Shall we add that also?

Reply:
Corrected.